srcpkgs/fluxbox/template

```diff
@@ -10,17 +11,21 @@ configure_args="--enable-nls
  ac_cv_lib_Xrandr_XRRQueryExtension=yes ac_cv_lib_fribidi_fribidi_version_info=yes"
 hostmakedepends="pkg-config"
 makedepends="libSM-devel libXrender-devel libXft-devel libXpm-devel imlib2-devel
- libXext-devel libXinerama-devel libXrandr-devel fribidi-devel"
+ libXext-devel libXinerama-devel libXrandr-devel fribidi-devel automake libtool
+ gettext gettext-devel"
 short_desc="Highly configurable and low resource X11 Window manager"
 maintainer="Orphaned <orphan@voidlinux.org>"
-homepage="http://www.fluxbox.org"
 license="MIT"
-distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
-checksum=fc8c75fe94c54ed5a5dd3fd4a752109f8949d6df67a48e5b11a261403c382ec0
+homepage="http://www.fluxbox.org"
+distfiles="https://github.com/fluxbox/fluxbox/archive/refs/tags/Release-${version//./_}.tar.gz"
+checksum=990414ae342a35c5703e5025a8bf019b0b4dada55d57071c51d3d498abb5cd36
 
 case "$XBPS_TARGET_MACHINE" in
 	*-musl) configure_args="${configure_args/enable-nls/disable-nls}" ;;
 esac
+pre_configure() {
+	./autogen.sh
+}
```

Comment:
nitpick: add an empty line between the esac, this function, and the next function